sw/banshee/Makefile

```
@@ -64,7 +66,7 @@ define test_template
 		LOGFILE=`mktemp`; \
 		if [ ! -z $(filter $(1),$(TESTS_BLACKLIST)) ]; then \
 			echo "$$CMD ... `tput setaf 3`ignored`tput sgr0`"; \
-		elif ! env SNITCH_LOG=debug $$CMD &> $$LOGFILE; then \
+		elif ! { env SNITCH_LOG=$(SNITCH_LOG) $$CMD; } $(TEST_SUFFIX); then \
```

Good idea! How about (might be a rebasing artifact):
```suggestion
		elif ! env SNITCH_LOG=$(SNITCH_LOG) $$CMD &> $$LOGFILE; then \
```